mod_reforged/hooks/skills/actives/throw_dirt_skill.nut

```diff
@@ -0,0 +1,36 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/throw_dirt_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		this.m.Description = "This character hides a substantial amount of sand in their pockets, ready to be ready to be thrown at someone.";
+	}
+
+	q.onVerifyTarget = @(__original) function( _originTile, _targetTile )
+	{
+		local ret = __original(_originTile, _targetTile);
+
+		if (ret && _targetTile.getEntity().getSkills().hasSkill("effects.distracted"))	// Throw Dirt can no longer be used one someone who is already affected by it
+		{
+			return false;
+		}
+		else
+		{
+			return ret;
+		}
```

Comment:
This is needlessly complicated to follow through logically with all the if/else. This can simply be the following. Please use this:
```squirrel
return __original(_originTile, _targetTile) && !_targetTile.getEntity().getSkills().hasSkill("effects.distracted");
```